<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Products in Transactions</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="filter_products_in_transactions_label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="filter_products_in_transactions_input">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="products_in_transactions_table">
          <column>
           <property name="text">
            <string>SKU</string>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Product Name</string>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Price</string>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Stock</string>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Unit</string>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Created At</string>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="close_products_in_transactions_button">
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_products_in_transactions_button">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
